Source/CDRStateTracker.m

```diff
@@ -0,0 +1,18 @@
+#import "CDRStateTracker.h"
+#import "CDRRunState.h"
+
+@implementation CDRStateTracker
+
+- (void)didStartBuildingTestTree {
```

Comment:
Similar to my comments about the enum; not sure the fact that we're building a tree structure is useful or relevant.